README.md

```diff
@@ -48,6 +48,13 @@ $
 
 I know right?
 
+### Usage
+
+Now that you have deployed nothing, you are ready to not use nocode:
+```
+$ rm -rf ../nocode
```

Comment:
Looks like code. This is not something we want, do we?

Reply:
To much code

Reply:
omg so much code